File: Api/MySerialList.Data/Model/Episode.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace MySerialList.Data.Model
{
    public class Episode
    {

        public int Id { get; set; }
        public string Title { get; set; }
        public string Released { get; set; }
        public int FilmProductionId { get; set; }
        public int Season { get; set; }
        public int EpisodeNumber { get; set; }

        [ForeignKey("FilmProductionId")]
        public virtual FilmProduction FilmProduction { get; set; }
        public virtual IEnumerable<ReviewFilmProduction> Reviews { get; set; }
    }
}

File: my_serial_list/lib/features/film_production/domain/entities/episode.dart
import 'package:meta/meta.dart';

class Episode {
  final String title;
  final String released;
  final int filmProductionId;
  final int season;
  final int episodeNumber;
  final bool watched;

  Episode({
    @required this.title,
    @required this.released,
    @required this.filmProductionId,
    @required this.season,
    @required this.episodeNumber,
    @required this.watched,
  });
}
